papers/david_dotson/david_dotson.rst

```diff
@@ -539,7 +540,7 @@ A ``Sim`` featues special limbs for storing an MDAnalysis_ ``Universe`` definiti
 As an example of effectively using ``Sims``, say we have 50 biased molecular dynamics simulations that sample the conformational change of the ion transport protein NhaA [Lee14]_ from the inward-open to outward-open state (Figure :ref:`fig:nhaa`).
 Let's also say that we are interested in how many hydrogen bonds exist at any given time between the two domains as they move past each other.
 
-We can use the MDAnalysis ``HydrogenBondAnalysis`` class to collect the data for each ``Sim`` using ``Bundle.map`` for process parallelism, and store it using the ``datreant.data`` limb:
+We can use the MDAnalysis ``HydrogenBondAnalysis`` class to collect the data for each ``Sim`` using ``Bundle.map`` for process parallelism, storing the results using the ``datreant.data`` limb:
```

Comment:
Decided to keep "storing". It's really minor, and I think it's the better one between the two.